stable/velero/README.md

```diff
@@ -82,6 +82,7 @@ Parameter | Description | Default
 `metrics.serviceMonitor.enabled` | Set this to `true` to create ServiceMonitor for Prometheus operator | `false`
 `metrics.serviceMonitor.additionalLabels` | Additional labels that can be used so ServiceMonitor will be discovered by Prometheus | `{}`
 `schedules` | A dict of schedules | `{}`
+`cleanUpCRDs` | Set this to `true` to delete any created CustomResourceDefintions and their associated instances. Backup representations in the cluster will be deleted, but data in object storage and volume snapshots are not. | `false`
```

Comment:
Perhaps we should intentionally leave this out of user-facing documentation? We could also remove from `values.yaml` and only leave a comment in the cleanup job template itself for power users to discover.

I can't think of any use cases outside of CI where offloading the CRD deletion decision to a cluster administrator or externally managed script wouldn't be preferable to having someone potentially setting and forgetting this value.

Reply:
Yeah, I'm fine with that, as this is largely a CI concern and not for end users.